cmd/peers.go

```diff
@@ -208,37 +213,67 @@ func (o *PeersOptions) Info() (err error) {
 
 // List shows a list of peers
 func (o *PeersOptions) List() (err error) {
+
+	// convert Page and PageSize to Limit and Offset
+	page := util.NewPage(o.Page, o.PageSize)
+
 	if o.Network == "ipfs" {
 		res := []string{}
-		if err := o.PeerRequests.ConnectedIPFSPeers(&o.Limit, &res); err != nil {
+		limit := page.Limit()
+		if err := o.PeerRequests.ConnectedIPFSPeers(&limit, &res); err != nil {
 			return err
 		}
 
 		for i, p := range res {
 			printSuccess(o.Out, "%d.\t%s", i+1, p)
 		}
 	} else {
-
 		// if we don't have an RPC client, assume we're not connected
-		if o.UsingRPC && !o.Cached {
+		if !o.UsingRPC && !o.Cached {
 			printInfo(o.Out, "qri not connected, listing cached peers")
 			o.Cached = true
 		}
 
 		p := &lib.PeerListParams{
-			Limit:  o.Limit,
-			Offset: o.Offset,
+			Limit:  page.Limit(),
+			Offset: page.Offset(),
 			Cached: o.Cached,
 		}
 		res := []*config.ProfilePod{}
 		if err = o.PeerRequests.List(p, &res); err != nil {
 			return err
 		}
 
+		// If the length of the peers list is greater then 25
+		// send the output to a terminal pager
+		if len(res) > 25 {
```

Comment:
This `less` stuff is _very_ cool. Glad to see you've only introduced it in a small place as an experiment. I think we should merge this & follow up with another PR that unifies all pagniated printing into a single function detects & uses `PAGER`.

Reply:
but while we're here, I don't think we need the arbitrary 25 distinction. let's just always print with this new codepath.